addons/survey/static/src/scss/survey_form.scss

```diff
@@ -0,0 +1,251 @@
+.o_survey_header {
+    height: 50px;
+}
+.o_survey_main {
+    min-height: calc(100% - 105px);
+    padding-top: 16px;
+}
+
+.o_survey_question_text_box,
+.o_survey_question_date,
+.o_survey_question_datetime,
+.o_survey_question_numerical_box {
+    border: 0px;
+    border-radius: 0px;
+    border-bottom: 2px solid $o-brand-primary;
+    background-color: transparent !important;
+    font-weight: bold;
+    color: $o-brand-primary !important;
+    &:disabled {
+        border-color: grey;
+        color: black !important;
+        border-bottom: 2px solid grey;
+    }
+    &:focus {
+        border-color: $o-brand-primary;
+    }
+
+    &::-webkit-scrollbar {
+      width: 10px;
+    }
+    &::-webkit-scrollbar-thumb {
+      background: grey;
+      border-radius: 10px;
+    }
+    &::-webkit-scrollbar-thumb:hover {
+      background: white;
+    }
+}
+
+.o_survey_form_date .input-group-append {
+    margin-left: -36px;
+    .input-group-text {
+        border: 0px;
+        background-color: transparent;
+        i {
+            color: $o-brand-primary;
+            font-size: large;
+        }
+    }
+}
+
+.o_survey_comment_container {
+    padding: 0rem 0.25rem;
+    &.col-lg-12 {
+        padding: 0rem 1.25rem;
+    }
+}
+
+.o_survey_choice_container {
+    flex-wrap: wrap;
+}
+
+.o_survey_choice_btn {
+    color: white;
+    background-color: rgba(0, 160, 157, 0.6);
+    transition: background-color 0.3s ease;
+    border: 2px solid $o-brand-primary;
+    border-radius: 3px;
+    cursor: pointer;
+    padding: 0.25rem 1rem;
+    margin: 0.25rem;
+    flex-grow: 1;
+    min-width: 300px;
+
+    span {
+        margin-left: 0.5rem;
+        line-height: 25px;
+    }
+    input {
+        visibility: hidden;
+        position: absolute;
+    }
+    i {
+        margin-top: 3px;
+        display: inline;
+        transition: display 0.3s ease;
+        position: relative;
+        top: 0px;
+        font-size: large;
+        &.fa-check-circle,&.fa-check-square {
+            top: -2px;
```

Comment:
negative top? :(
`bottom: 2px` doesn't do the trick?

Reply:
nope.. bah, I give up ! :(